### PR TITLE
Style single event page

### PR DESCRIPTION
**Pull request by Suuuuha**
related to #124 

Note: since the map PR is not merged yet i used a picture from the framework instead of the map for now.
screenshot from 2017-07-31 13 52 55
screenshot from 2017-07-31 13 53 06

### CODE REVIEW

File: src/views/event.hbs [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
-      <p>{{this.event.eventDescription}}</p>
-    </div>
-    <div class="">
-      <a href="#">RSVP</a>
-      <span>{{this.event.eventDate}}</span>
-      <span>{{this.event.eventTime}}</span>
-    </div>
+<div class="cf ph6 ph6-ns pv6 ml5">
+  <div class="fl-ns w-80-ns pr4-ns">
+    <h1 class="mb3 mt0 lh-title mid-gray">{{ this.event.eventTitle}}</h1>
```
Comment (Karyum) [inline src/views/event.hbs]: so travis is failing cause this has changed, what do i mean is in the tests its like this :https://github.com/UKITH/Arabs-Startups/blob/master/tests/server/index.js#L143 
so in this PR change the test to be without the h1 tag

**Comment by Suuuuha** [comment]
@Karyum fixed the tests

File: src/views/event.hbs [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
+      <span id="organiser" class="lh-copy tc">{{this.event.eventOrganiser}}</span>
+      <address class="lh-copy pt5 ">{{this.event.eventAddress}}</address>
     </div>
   </div>
-</div>
+</div
 
 <script type="text/javascript" src="/map.js"></script>
-<script src={{{getMapLink}}}></script>
+<!-- <script src={{{getMapLink}}}></script> -->
```
Comment (esraajb) [inline src/views/event.hbs]: remove?

**Comment by Suuuuha** [comment]
@esraajb requested changes are done :)

**Pull request merged**